Cargo.toml

```diff
@@ -83,6 +92,13 @@ api-explorer = [
 api-node = [
     "api",
 ]
+# To use this feature, you must also enable the unstable cfg like so:
+# > RUSTFLAGS="--cfg tokio_unstable" cargo build --features console
+# > cargo build --profile console --features console
+console = [
+    "dep:console-subscriber",
+    "tokio/tracing",
+]
```

Comment:
Maybe I am doing something stupid, but using the `2nd` option doesn't compile for me when it tries to compile `tokio-1.18.0` with error `use of undeclared crate or module tracing`.